### PR TITLE
chore(docs) Adding IN to FAQ

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: docs/troubleshooting/faq.md [hunk 1]
```diff
@@ -88,3 +88,19 @@ We have an open
 inserted with identical fields. Until then, you need to
 [modify the data](/docs/guides/modifying-data) after it's inserted and use a
 `GROUP BY` query to identify duplicates.
+
+## How do I query recursively within a time range?
```
Comment (jerrinot) [inline docs/troubleshooting/faq.md:92]: nit: is it possible to use simpler language for the title? the question we keep getting is this: "Can we query by time?" "querying recursively within a time range" is a rather complex sentence to parse.

**Review by jerrinot — DISMISSED** [review]
I'd personally use simpler language, but that could be just a personal taste.

File: docs/troubleshooting/faq.md (hunk 1 above)
Comment (amyshwang) [inline docs/troubleshooting/faq.md:92]:
```suggestion
## Can I query by time?
```

File: docs/troubleshooting/faq.md [hunk 2]
```diff
@@ -88,3 +88,19 @@ We have an open
...
+
+When using the `WHERE` statement to define the time range for a query, the
```
Comment (amyshwang) [inline docs/troubleshooting/faq.md:94]:
```suggestion
Yes! When using the `WHERE` statement to define the time range for a query, the
```

File: docs/troubleshooting/faq.md (hunk 1 above)
Comment (amyshwang) [reply docs/troubleshooting/faq.md]: Good point - updated!

**Review by jerrinot — APPROVED** [review]
now it's pretty awesome: simple and straight to the point. 
thank you!

**Pull request merged**